backend/src/packages/chaiNNer_standard/image_dimension/utility/get_bounding_box.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import annotations
+
+from typing import Tuple
+
+import numpy as np
+
+from nodes.properties.inputs import ImageInput, SliderInput
+from nodes.properties.outputs import NumberOutput
+from nodes.utils.utils import get_h_w_c
+
+from .. import utility_group
+
+
+@utility_group.register(
+    schema_id="chainner:image:get_bbox",
+    name="Get Bounding Box",
+    description="Gets a bounding box (X, Y, Height, and Width) of the white area of a mask.",
+    icon="BsRulers",
+    inputs=[
+        ImageInput(channels=1),
+        SliderInput(
+            "Threshold",
+            precision=1,
+            minimum=1,
```

Comment:
Why is the minimum 1? 

Threshold=0 would mean that the bounding box is the entire image, right? If so, then it should be allowed. Having neutral values can be very useful. E.g. the Math node doesn't forbid addition with 0 just because it doesn't do anything.

Reply:
That was a remnant of when I was tinkering with cv2's actual system for thresholds. I'll change that to 0.